testnets/iopntestnet/assetlist.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "iopntestnet",
+  "assets": [
+    {
+      "description": "Native token of Iopn Chain",
+      "denom_units": [
+        {
+          "denom": "gwei",
+          "exponent": 0
+        },
+        {
+          "denom": "gwei",
+          "exponent": 5
+        }
+      ],
+      "base": "gwei",
+      "name": "Iopn Chain",
+      "display": "iopn",
+      "symbol": "OPN",
+      "logo_URIs": {
+        "png": "https://f005.backblazeb2.com/file/tracehawk-prod/logo/IOPn/Light.png"
```

Comment:
Images must be uploaded into an /images/ directory inside this chain's directory in this repo.